common/test/test_migrations/utils.py

```diff
@@ -0,0 +1,57 @@
+"""
+General class for Test migrations.Based off an implementation provided at
```

Comment:
typo: `Test` should be lowercase, and there should be a space between `migrations.` and `Based`